### ISSUE TITLE
Adding an Installation Guide for Qubes OS on Virtual Machines

### ISSUE THREAD

**Issue by zxcv2008a** [QubesOS/qubes-issues#5954]
**The problem you're addressing (if any)**
The Qubes OS is missing an installation guide for virtual machines. Users are installing an outdated and unsupported version of Qubes OS (3.2) instead of the supported version. some unofficial installation guide for Qubes OS on a virtual machine is misleading and needed documentation. reference "https://www.youtube.com/watch?v=mATI8Lht0Js". Usually, users face some errors and mistakes while installing Qubes OS on virtual machine. reference "https://www.virtualbox.org/ticket/16771"

**Describe the solution you'd like**
I would like to add an installation guide for Qubes OS using supported Virtual machines on the website.

**Where is the value to a user, and who might that user be?**
Normal Users who would like to try Qubes OS on a virtual machine without installing it in their system. Advanced users that would like to use Qubes OS as a security lab.

**Describe alternatives you've considered**
Adding an installation guide on a virtual machine would give a unified and user-friendly installation guide.

**Relevant [documentation](https://www.qubes-os.org/doc/) you've consulted**
https://www.qubes-os.org/doc/installation-guide/

**Comment by zxcv2008a** [comment]
#1021

**Comment by andrewdavidwong** [comment]
It looks like @zxcv2008a also submitted a PR for this: https://github.com/QubesOS/qubes-doc/pull/1021

**Comment by zxcv2008a** [comment]
I'm the one who submitted this issue.

**Comment by andrewdavidwong** [comment]
> I'm the one who submitted this issue.

Yes, I know. I was adding the information that you both submitted the PR and opened this issue, and I linked them together, since you did not mention or link to your PR here (or vice versa in the PR). This way, everyone else can easily discover that the issue and PR both exist and go together.

**Comment by zxcv2008a** [comment]
Thank you

**Comment by zxcv2008a** [comment]
> > I'm the one who submitted this issue.
> 
> Yes, I know. I was adding the information that you both submitted the PR and opened this issue, and I linked them together, since you did not mention or link to your PR here (or vice versa in the PR). This way, everyone else can easily discover that the issue and PR both exist and go together.

Thank you

**Comment by andrewdavidwong** [comment]
https://github.com/QubesOS/qubes-doc/pull/1021 was merged a long time ago.

Closing this as resolved. If you believe the issue is not yet resolved, or if anyone is still affected by this issue, please leave a comment, and we'll be happy to reopen this. Thank you.

**Comment by andrewdavidwong** [comment]
> [QubesOS/qubes-doc#1021](https://github.com/QubesOS/qubes-doc/pull/1021) was merged a long time ago.

Oh wait. That PR was just to add it as a GSoD proposal.

**Comment by andrewdavidwong** [comment]
Related to/possible duplicate of #5041, #2249

**Comment by ben-grande** [comment]
> Related to/possible duplicate of: [#5041](https://github.com/QubesOS/qubes-issues/issues/5041), [#2249](https://github.com/QubesOS/qubes-issues/issues/2249).

@andrewdavidwong  I believe it is duplicate and should be closed.
